v2/build/collegevine/standalone.txt

```diff
@@ -0,0 +1,62 @@
+core/Seadragon.js
+core/Point.js
+core/Size.js
+core/Rect.js
+utils/String.js
+utils/Debug.js
+utils/Object.js
+utils/Function.js
+utils/Browser.js
+utils/Math.js
+utils/Math_clamp.js
+utils/Math_log.js
+utils/Math_morton.js
+utils/Math_round.js
+utils/Uri.js
+utils/Xml.js
+utils/Xml_fetch.js
+utils/Xml_parse.js
+utils/Element.js
+utils/Event.js
+utils/Event_createRaise.js
+utils/Event_cancelStop.js
+utils/Event_addRemove.js
+utils/Timer.js
+utils/EventManager.js
+utils/Mouse.js
+utils/Page.js
+image/TileCache.js
+image/TileInfo.js
+image/TileSource.js
+image/DisplayRect.js
+image/DziTileSource.js
+image/DzcTileSource.js
+image/DeepZoom.js
+image/Network.js
+image/Tile.js
+image/Level.js
+image/TileLoader.js
+image/ImageManager.js
+image/Drawer.js
+image/CanvasDrawer.js
+image/ImgDrawer.js
+image/ImageState.js
+image/Image.js
+zoomcontainer/MouseTracker.js
+zoomcontainer/Spring.js
+zoomcontainer/SVGZoomContainer.js
+zoomcontainer/HTMLZoomContainer.js
+zoomcontainer/CanvasZoomContainer.js
+zoomcontainer/Viewport.js
+zoomcontainer/Viewer.js
+../app/pivot/Pivot.js
+../app/pivot/Number.js
+../app/pivot/NumberPicker.js
+../app/pivot/Date.js
+../app/pivot/DatePicker.js
+../app/pivot/Slider.js
+../app/pivot/Template.js
+../app/pivot/Lzw.js
+../app/pivot/PivotViewer.js
+../app/pivot/CxmlLoader.js
+../app/collegevine/App.js
```

Comment:
Old school build manifest that says which files and in what order they should be concatenated. Ideally, this would be replaced with self-documenting `import` statements + Webpack bundling, but that’s _far_ out of scope for my work.